easybuild/easyconfigs/p/Python/Python-3.11.3-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,73 @@
+name = 'Python'
+version = '3.11.3'
+
+homepage = 'https://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+checksums = ['1a79f3df32265d9e6625f1a0b31c28eb1594df911403d11f3320ee1da1b3e048']
+
+builddependencies = [
+    ('UnZip', '6.0'),
+    ('pkgconf', '1.9.5'),
+]
+
+dependencies = [
+    ('binutils', '2.40'),
+    ('bzip2', '1.0.8'),  # required for bz2 package in Python stdlib
+    ('zlib', '1.2.13'),
+    ('libreadline', '8.2'),
+    ('ncurses', '6.4'),
+    ('SQLite', '3.42.0'),
+    ('XZ', '5.4.2'),
+    ('libffi', '3.4.4'),
+    ('OpenSSL', '1.1', '', SYSTEM),
+]
+
+install_pip = True
+
+exts_default_options = {
+    'download_dep_fail': True,
+    'sanity_pip_check': True,
+    'source_urls': [PYPI_SOURCE],
+    'use_pip': True,
+}
+
+# order is important!
+# package versions updated 2023-05-17
+exts_list = [
+    ('flit_core', '3.9.0', {
+        'checksums': ['72ad266176c4a3fcfab5f2930d76896059851240570ce9a98733b658cb786eba'],
+    }),
+    ('wheel', '0.40.0', {
+        'checksums': ['cd1196f3faee2b31968d626e1731c94f99cbdb67cf5a46e4f5656cbee7738873'],
+    }),
+    ('setuptools', '67.7.2', {
+        'checksums': ['f104fa03692a2602fa0fec6c6a9e63b6c8a968de13e17c026957dd1f53d80990'],
+    }),
+    ('pip', '23.1.2', {
+        'checksums': ['0e7c86f486935893c708287b30bd050a36ac827ec7fe5e43fe7cb198dd835fba'],
+    }),
+    ('distlib', '0.3.6', {
+        'checksums': ['14bad2d9b04d3a36127ac97f30b12a19268f211063d8f8ee4f47108896e11b46'],
+    }),
+    ('filelock', '3.12.0', {
+        'source_tmpl': SOURCE_PY3_WHL,
+        'checksums': ['ad98852315c2ab702aeb628412cbf7e95b7ce8c3bf9565670b4eaecf1db370a9'],
+    }),
+    ('platformdirs', '3.5.1', {
+        'source_tmpl': SOURCE_PY3_WHL,
+        'checksums': ['e2378146f1964972c03c085bb5662ae80b2b8c06226c54b2ff4aa9483e8a13a5'],
+    }),
+    ('virtualenv', '20.23.0', {
+        'source_tmpl': SOURCE_PY3_WHL,
+        'checksums': ['6abec7670e5802a528357fdc75b26b9f57d5d92f29c5462ba0fbe45feacc685e'],
+    }),
```

Comment:
Let's remove `virtualenv` and required dependencies, and keep this to the bare minimum (`setuptools` + `pip` + `wheel`, and `flit_core` because that's a required dependency for one of those).

```suggestion
```